plugin/plugins/postgresds/postgresds.go

```diff
@@ -0,0 +1,110 @@
+package postgresds
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	postgresdb "gx/whyrusleeping/QmXyfyKbv3EkZUtN5oxL2c32HNPaJKYsCQhxPufMPxKwzi/sql-datastore/postgres"
```

Comment:
shouldn't this be `gx/ipfs/QmXyfyKbv3EkZUtN5oxL2c32HNPaJKYsCQhxPufMPxKwzi/sql-datastore/postgres`?